platform-packaging/src/Platform/Packaging.hs

```diff
@@ -11,39 +11,115 @@ import "base"           Data.Typeable (Typeable)
 import "base"           Data.Data (Data)
 import "base"           Data.Semigroup (Semigroup, (<>))
 import "lens"           Control.Lens hiding (from)
+import "aeson"          Data.Aeson (ToJSON, FromJSON, toEncoding, genericToEncoding, defaultOptions)
 import "data-default"   Data.Default (Default, def)
 import "containers"     Data.Map.Strict (Map, empty, assocs)
 import "text"           Data.Text (Text)
-import "dockerfile"     Data.Docker (Docker, from, run, cmd)
+import "dockerfile"     Data.Docker
 import "platform-types" Platform.Types
 
+data OS = Ubuntu
```

Comment:
shouldnt types from here, move to platform-packaging-types?

Reply:
I've now moved the types.